app/views/merge_requests/details.html.erb

```diff
@@ -0,0 +1,64 @@
+<% content_for :before_content do %>
+  <% title = "Merge details: #{@merge_request.absorbing_organisation_name}" %>
+  <% content_for :title, title %>
+  <%= govuk_back_link href: "#{organisations_path}#merge-requests" %>
+<% end %>
+<% unless @merge_request.signed_dsa || @merge_request.absorbing_organisation_id.blank? %>
+  <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+    <div class="govuk-notification-banner__header">
+      <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+        Important
+      </h2>
+    </div>
+    <div class="govuk-notification-banner__content">
+      <strong>The absorbing organisation must accept the Data Sharing Agreement before merging.</strong>
+      <br>
+      <% if @merge_request.dpo_user %>
+        Contact the Data Protection Officer: <%= link_to @merge_request.dpo_user.name, user_path(@merge_request.dpo_user.id) %>
+      <% else %>
+        <%= @merge_request.absorbing_organisation_name %> does not have a Data Protection Officer. You can assign one on the <%= link_to 'users page', "#{organisation_path(@merge_request.absorbing_organisation_id)}/users" %>.
+      <% end %>
+    </div>
+  </div>
+<% end %>
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l">Merge details</span>
+  <%= @merge_request.absorbing_organisation_name %>
+</h1>
+<% unless @merge_request.status == "request_merged"%>
+<div class="govuk-button-group">
+  <button type="submit" <%= @merge_request.status == "ready_to_merge" ? "" : "disabled aria-disabled=\"true\"" %> class="govuk-button" data-module="govuk-button">
+    Begin merge
+  </button>
+  <button type="submit" class="govuk-button govuk-button--warning" data-module="govuk-button">
+    Delete merge request
+  </button>
+</div>
+<% end %>
+
+<% request_details = [
+  { label: "Requester", value: display_value_or_placeholder(@merge_request.requesting_user&.name) },
+  { label: "Helpdesk ticket", value: @merge_request.helpdesk_ticket.present? ? link_to("#{@merge_request.helpdesk_ticket} (opens in a new tab)", "https://dluhcdigital.atlassian.net/browse/#{@merge_request.helpdesk_ticket}", target: "_blank", rel: "noopener noreferrer") : display_value_or_placeholder(nil), action: @merge_request.status == "request_merged" ? nil : { text: "Change", href: "#", visually_hidden_text: "helpdesk ticket" } },
+  { label: "Status", value: status_tag(@merge_request.status) },
+] %>
```

Comment:
Each question/detail table is built according to this spec which determines the name of the row (label); the value and the accessibility hidden text for the action buttons/links.

Reply:
Moved it to the helper